doc/Command-Reference.md

```diff
@@ -4923,6 +4924,40 @@ Optionally, you can specify a period (in seconds) with which to gather counters
   admin@sonic:~$ sonic-clear rifcounters
   ```
 
+The "fec-histogram" subcommand is used to display the fec histogram for the port.
+
+When data is transmitted, it's broken down into units called codewords. FEC algorithms add extra data to each codeword that can be used to detect and correct errors in transmission.
+In a FEC histogram, "bins" represent ranges of errors or specific categories of errors. For instance, Bin0 might represent codewords with no errors, while Bin1 could represent codewords with a single bit error, and so on. The histogram shows how many codewords fell into each bin. A high number in the higher bins might indicate a problem with the transmission link, such as signal degradation.
+
+- Example:
+  ```
+  admin@str-s6000-acs-11:/usr/bin$ show interface counters fec-histogram -i <PORT>
+
+Symbol Errors Per Codeword  Codewords
+--------------------------  ---------
+BIN0:                       1000000
+BIN1:                       900000
+BIN2:                       800000
+BIN3:                       700000
+BIN4:                       600000
+BIN5:                       500000
+BIN6:                       400000
+BIN7:                       300000
+BIN8:                       0
+BIN9:                       0
+BIN10:                      0
+BIN11:                      0
+BIN12:                      0
+BIN13:                      0
+BIN14:                      0
+BIN15:                      0
+BIN16:                      0
```

Comment:
@vdahiya12 do we have 17 bins or 16 bins?

Reply:
fixed, 16 only